src/api/forms/repositories/form-definition-repository.js

```diff
@@ -597,6 +597,52 @@ export async function addLists(
 
   return lists
 }
+
+/**
+ * Updates a Draft Form list by id
+ * @param {string} formId
+ * @param {string} listId
+ * @param {List} listItem
+ * @param {ClientSession} session
+ * @param {FormStatus} [state]
+ * @returns {Promise<List>}
+ */
+export async function updateList(
+  formId,
+  listId,
+  listItem,
+  session,
+  state = FormStatus.Draft
+) {
+  if (state === FormStatus.Live) {
```

Comment:
The `state` arg and guard `if (state === FormStatus.Live)` seem a bit redundant maybe?
The function is described as "Updates a Draft Form list" - so it should only ever act on `draft.lists`